src/Core/src/Handlers/WebView/WebViewHandler.Windows.cs

```diff
@@ -344,14 +348,15 @@ void OnWindowClosed(object sender, WindowEventArgs args)
 				}
 			}
 
-			void OnCoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs args)
+			async void OnCoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs args)
 			{
 				sender.CoreWebView2.HistoryChanged += OnHistoryChanged;
 				sender.CoreWebView2.NavigationStarting += OnNavigationStarting;
 				sender.CoreWebView2.NavigationCompleted += OnNavigationCompleted;
 
-				if (Handler is WebViewHandler handler)
+				if (Handler is WebViewHandler handler && handler.VirtualView is not null)
 				{
+					await handler.SyncPlatformCookiesToVirtualView(sender.CoreWebView2.Source);
```

Comment:
Can we move this under "UpdateUserAgent" and then use our `.FireAndForget` extension to avoid the "async void" ?

Reply:
I’ve moved the SyncPlatformCookies method under the UpdateUserAgent method. Could you please review the changes?